lib/dir_model/import.rb

```diff
@@ -45,22 +45,21 @@ def next(path, context={}, previous=nil)
 
     def match?
       return if load_state == :loaded
+      @_match = loader { find_match }
```

Comment:
In `find_match` I store if there is a match, true or false in `@_match` and all informations about matches into `@file_infos` so in `skip?` I use :  

```
    def skip?
      load
      !@_match
    end
```
and in `matches` 
```
    def matches
      load
      file_infos[:options][:match]
    end
```